package.json

```diff
@@ -37,9 +37,9 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
     "chai-sinon": "^2.8.1",
-    "coveralls": "^2.12.0",
+    "coveralls": "^3.1.1",
     "istanbul": "^0.4.5",
-    "mocha": "^3.2.0",
+    "mocha": "^9.2.2",
```

Comment:
This change + the lockfile change should probably be reverted, to provide a cleaner diff for @sjurba to review in the target PR.